mdata/store_cassandra.go

```diff
@@ -435,12 +449,20 @@ func (c *CassandraStore) SearchTable(key, table string, start, end uint32) ([]ch
 	results := make(chan outcome, numQueries)
 	for i := range crrs {
 		crrs[i].out = results
-		c.readQueue <- crrs[i]
+		select {
+		case c.readQueue <- crrs[i]:
+		default:
+			cassReadQueueFull.Inc()
+			return nil, errReadQueueFull
+		}
```

Comment:
switching from block-on-full to drop-on-full has some ramifications. consider the setting `cassandra-read-queue-size`, currently defaulting to 100 and documented as "max number of outstanding reads before blocking. value doesn't matter much"

this value will need to be increased significantly, and clearly the consequences of this setting are now different, and the value needs to be documented differently.